x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/get_signals_template.ts

```diff
@@ -97,6 +112,47 @@ export const createSignalsFieldAliases = () => {
   return fieldAliases;
 };
 
+export const backwardsCompatibilityMappings = [
+  {
+    minVersion: 0,
+    // Version 45 shipped with 7.14
+    maxVersion: 45,
```

Comment:
is there a central point we can pull this number from so we don't have to change it piecewise every update?

Reply:
This number should never change, as any future indices won't need this migration. The next migration will be a separate object and may have a different `maxVersion`, but we'll leave this migration the same.